contracts/Core/StakeManager.sol

```diff
@@ -297,12 +297,13 @@ contract StakeManager is Initializable, ACL, StakeStorage {
     /// @param id The ID of the staker who is penalised
     /// @param bountyHunter The address of the bounty hunter
     function slash (uint256 id, address bountyHunter, uint256 epoch) external onlyRole(parameters.getStakeModifierHash()) {
-        uint256 halfStake = stakers[id].stake/(2);
-        _setStakerStake(id, 0, "Slashed", epoch);
-        if (halfStake > 1) {
-            require(sch.transfer(bountyHunter, halfStake), "failed to transfer bounty");
+      uint256 halfStake = ((stakers[id].stake)*(parameters.percentSlash()))/(2*100);
+      uint256 Stake =  stakers[id].stake - ((stakers[id].stake*parameters.percentSlash())/100);
```

Comment:
I think percentSlash is not clear
slashPenalty ?

Reply:
Again here only addition should be changing to governance param, please remove lint changes

Reply:
@abhishekvispute it is how much percent of the stake should we slash of the staker so it can be percentSlashPenalty ?